db/schema.rb

```diff
@@ -10,11 +10,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_12_07_173240) do
+ActiveRecord::Schema.define(version: 2019_12_07_182957) do
```

Comment:
What happened here? Why was the schema version changed?

Reply:
I had to create the game model to represent playing a game. I created the game model based off the spreadsheet we have going. I created user_id and id. If we want to hold off on this until the game model is totally ready that's fine. We can talk about it on Wednesday.